src/index.html

```diff
@@ -885,6 +889,7 @@ <h3><wvre-text value="Second Tab Header"></wvre-text></h3>
     }
   }"></wvre-theme>
 
+  <script defer src="tinymce/tinymce.min.js"></script>
```

Comment:
We need to avoid the global usage of tinymce.

Reply:
Removed this import